Dockerfile

```diff
@@ -7,47 +7,47 @@ RUN apk --no-cache add ca-certificates \
 
 RUN mkdir /source
 WORKDIR /source
-RUN curl -Lo strichliste.tar.gz https://github.com/strichliste/strichliste/releases/download/v1.6.2/strichliste.tar.gz
+RUN curl -Lo strichliste.tar.gz https://github.com/strichliste/strichliste/releases/download/v1.8.2/strichliste-v1.8.2.tar.gz
 RUN tar -xf strichliste.tar.gz
 RUN rm -r strichliste.tar.gz
 
 
-FROM alpine:3.8
+FROM alpine:3.18.3
 
 RUN apk --no-cache add ca-certificates \
   && apk --no-cache add \
     curl \
-    php7 \
-    php7-ctype \
-    php7-tokenizer \
-    php7-iconv \
-    php7-mbstring \
-    php7-xml \
-    php7-json \
-    php7-dom \
-    php7-pdo_mysql \
-    php7-fpm \
+    php81 \
+    php81-ctype \
+    php81-tokenizer \
+    php81-iconv \
+    php81-mbstring \
+    php81-xml \
+    php81-json \
+    php81-dom \
+    php81-pdo_mysql \
+    php81-fpm \
+    php81-session \
```

Comment:
Could you also add `php81-sqlite3` and `php81-pdo_sqlite` so that `php bin/console app:import ...` works?